config.yml

```diff
@@ -8,7 +8,7 @@ defaults:
     rabbitQueueExpirationDelay: '- 24 hours' # This means we re-send alerts at most once a day
 
     # namespaces are rotated at this time interval.
-    namespaceRotationInterval: '1 hour'
+    namespaceRotationInterval: '6 hours'
```

Comment:
If we don't disconnect, then there's no point in doing any of this -- which might be the case, since this was largely invented as an intern project! In that case, any credentials are effectively lifetime credentials, as they can be used forever (as long as the TCP connection doesn't break). Since we rarely turn over connections (about once a day in Heroku, less or never in docker-cloud), the DNS load-balancing doesn't have much opportunity to adjust to changing circumstances.  And we will likely never see our reconnect logic used in production until a CloudAMQP node crashes and we go around manually clicking "Restart All Dynos" on every Heroku app.

What you're suggesting is a very different design.  We've already put a lot of time into "finishing up" what is ultimately an unnecessary service.  If we're now going to redesign it yet again, I'd rather abandon the whole effort and delete the repo.